### PR TITLE
PythonRandomVector improvement

### PR DESCRIPTION
**Pull request by josephmure**
The following methods can be implemented by `PythonRandomVector`:

- `setParameter`
- `getParameter`
- `getParameterDescription`

The `PythonRandomVector` doc page is trimmed with unimplemented methods removed.

### CODE REVIEW

**Review by jschueller — APPROVED** [review]

File: python/src/PythonRandomVector.cxx [hunk 1]
```diff
@@ -249,6 +249,67 @@ Bool PythonRandomVector::isEvent() const
...
+    if (callResult.isNull())
+    {
+      handleException();
+    }
+    Point parameter(convert< _PySequence_, Point >(callResult.get()));
+    return parameter;
+  }
+  else
+  {
+    // DistributionImplementation::getParameter throws
```
Comment (regislebrun) [inline python/src/PythonRandomVector.cxx:269]: Why do you mention DistributionImplementation in the comment?

**Review by regislebrun — APPROVED** [review]

File: python/src/PythonRandomVector.cxx (hunk 1 above)
Comment (josephmure) [reply python/src/PythonRandomVector.cxx:269]: I don't know! I did not write these lines, I simply copied them from [PythonDistribution.cxx](https://github.com/josephmure/openturns/blob/03980409340dee0d2e7f56a1821f97cb74dac740/python/src/PythonDistribution.cxx#L929).

Comment (josephmure) [reply python/src/PythonRandomVector.cxx:269]: @jschueller authored the lines back in PR #612

Comment (regislebrun) [reply python/src/PythonRandomVector.cxx:269]: When you copy/paste code, you also have to check comments ;-)

**Pull request merged**